com.chartboost.mediation/Runtime/Results/ChartboostMediationFullscreenAdLoadResult.cs

```diff
@@ -1,6 +1,6 @@
 using Chartboost.AdFormats.Fullscreen;
 
-namespace Chartboost.Requests
+namespace Chartboost.Results
```

Comment:
Hmmmm @kushG will this possibly break existing integrations?

Reply:
I think request and results should be part of the same namespace of `Requests`. 🤔  since results cannot exists without the first.